### ISSUE TITLE
Pain at the lateral aspect of the hip

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10824]
**Preferred term label:**
Pain at the lateral aspect of the hip

**Synonyms**
Lateral hip pain; trochanteric pain; pain on the outside of the hip

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Greater trochanteric pain syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001158
- label: Trochanteric pain
- parents: [Hip pain (HP:0030838)](https://hpo.jax.org/app/browse/term/HP:0030838)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the structures around the greater trochanter (the bony prominence on the outer part of the femur).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10824
- synonyms: Lateral hip pain(NONE)|Pain on the outside of the hip(NONE)|Pain at the lateral aspect of the hip(NONE)
